docs/architecture/clients/index.md

```diff
@@ -15,13 +15,55 @@ The mono-repository root directory contains three main folders.
 - `bitwarden_license` - Bitwarden Licensed version of the web vault.
 - `libs` - Shared code between the different applications.
 
-`libs` contains the following projects.
+## Libs
 
-- `Common` - Common code shared between all the clients including CLI.
-- `Angular` - Angular specific code used by all the visual clients.
-- `Components` - Angular Components Library.
-- `Node` - Used to be shared code for CLI and Directory Connector CLI, but since directory connector
+Historically, much of the feature logic of Bitwarden has existed within the `apps` directories, only
+being moved to `libs` (usually `libs/common` or `libs/angular`) if it needed to be shared between
+multiple clients.
+
+We are attempting to move to a more modular architecture by creating additional libs that are more
+feature-focused and more accurately convey team code ownership.
+
+- `common` - Common code shared between all the clients including CLI.
+- `angular` - Low-level Angular specific code used by all the visual clients. Code that is more tied
+  to Bitwarden business logic should be moved to a team-owned lib.
+- `node` - Used to be shared code for CLI and Directory Connector CLI, but since directory connector
   no longer uses the same version of libs this module is deprecated.
+- `admin-console` - Code owned by the Admin Console team.
+- `auth` - Code owned by the Auth team.
+- `billing` - Code owned by the Billing team.
+- `components` - Angular implementation of the Bitwarden design system.
+  [See more.](https://components.bitwarden.com/)
+- `exporter` - Code related to exporting; owned by the Tools team.
+- `importer` - Code related to importing; owned by the Tools team.
+- `platform` - Code owned by the Platform team.
+- `vault` - Code owned by the Vault team.
+
+For more on this approach, check out the
+[Nx documentation](https://nx.dev/concepts/more-concepts/applications-and-libraries):
+
+> place 80% of your logic into the `libs/` folder and 20% into `apps/`
+
+We are doing the opposite of this right now. What code should be moved from `apps/` to `libs/`? At
+this time, any code in the `/apps/web/*` that doesn't have a tight coupling with the web client can
+be moved into a lib. For components, this means they should only be the CL, no global CSS or
```

Comment:
⚠️ Unclear text:
> For components, this means **they should only be the CL**, no global CSS or
anything exported from web-specific modules.

Are you saying something like: "For components, they should only be *in* the component library with no global CSS or anything exported from web-specific modules"

Reply:
I rewrote this section in [8ce7a11](https://github.com/bitwarden/contributing-docs/pull/259/commits/8ce7a11d3657b49f87303d5efcf2df87e1c6111f), lemme know if this is better:

> We are doing the opposite of this right now. The code that remains in `apps/` should be primarily
> concerned with bootstrapping the application and consuming and configuring code exported from
> `libs/`. Over time, code not meeting this criteria should be moved from `apps/` to `libs/`.
> 
> Any code in the `apps/` that doesn't have a tight coupling with the client can be moved into a lib:
> if an exported member is wholly dependent on other libs but not `apps/`, it can likely be moved to a
> lib itself. (More concretely, this bars components that rely on global client-specific CSS, but not
> components built with Tailwind and the CL.)